src/block/VanillaBlocks.php

```diff
@@ -550,6 +550,7 @@
  * @method static Wood STRIPPED_OAK_WOOD()
  * @method static Log STRIPPED_SPRUCE_LOG()
  * @method static Wood STRIPPED_SPRUCE_WOOD()
+ * @method static StructureVoid STRUCTURE_VOID()
```

Comment:
Please do not update this doc block manually to ensure automatic method detection/generation functions properly.

Reply:
Technically speaking, while the docblock should indeed be generated, it's not of great importance, since the CI will verify that the docblock matches what's in the registry anyway. https://github.com/pmmp/PocketMine-MP/actions/runs/3521719774/jobs/5903887073